.github/workflows/neon-preview-branches-for-pull-requests.yml

```diff
@@ -30,6 +30,15 @@ jobs:
     name: Create/Delete Branch for Pull Request Job
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch integration data
+        shell: bash
+        run: |
+          WORKFLOW=$(curl -H "accept: application/json" -H "Authorization: Bearer ${{ secrets.NEON_API_KEY }}" https://console.neon.tech/api/v2/projects/${{ inputs.project_id }}/applications/github | jq -r ".workflows[] | select(.type == \"pr_branches\")")
```

Comment:
What if the request will fail?

Reply:
The PR is in draft.

Reply:
What if the request will fail, then pipeline should fail.
added `--fail-with-body -sSL` 
```
--fail-with-body
              (HTTP) Return an error on server errors where the HTTP response code is 400 or greater). In normal cases when an HTTP server fails to deliver a document, it returns an HTML document stating so
              (which often also describes why and more). This flag allows curl to output and save that content but also to return error 22.

              This is an alternative option to -f, --fail which makes curl fail for the same circumstances but without saving the content.
```